frontend/src/v5/ui/routes/viewer/tickets/ticketsForm/ticketGroups/groups/groups.component.tsx

```diff
@@ -39,9 +38,9 @@ export const Groups = ({ indexedOverrides, level = 0 }: GroupsProps) => {
 	const [overrideItems, overrideBatches] = partition(indexedOverrides, (o) => (o.prefix?.length || 0) === level);
 	const overridesByPrefix = groupBy(overrideBatches, (o) => o.prefix[level]);
 
-	const handleCheckboxClick = (e, prefix: string[]) => {
+	const handleCheckboxClick = (e, overrides) => {
 		e.stopPropagation();
-		toggleCollectionState(prefix);
+		toggleCollectionState(overrides.map(({ index }) => index));
```

Comment:
Can you add a small comment stating the overrides passed are all the overrides contained in the collection? (including descendants)